backend/src/agent/persistence/worker/persistence_worker_test.go

```diff
@@ -68,12 +68,13 @@ func TestPersistenceWorker_Success(t *testing.T) {
 	// Set up peristence worker
 	saver := NewWorkflowSaver(workflowClient, pipelineClient, 100)
 	eventHandler := NewFakeEventHandler()
-	worker := NewPersistenceWorker(
+	worker, err := NewPersistenceWorker(
 		util.NewFakeTimeForEpoch(),
 		"PERSISTENCE_WORKER",
 		eventHandler,
 		false,
 		saver)
+	assert.NoError(t, err)
```

Comment:
Can we change it to `require.NoError(t, err)`? It seems like there's no point in continuing the execution in case of error.
Can you do that for the other assertions too?